app/elements/socobo-ranking/socobo-ranking.html

```diff
@@ -542,7 +546,7 @@
          */
         _reload: function() {
           // notify user
-          this.notify("Reloading...");
+          this.notify("Ranking - Reloading...");
```

Comment:
Should be enough to display the toast only on clicking the reload button since the user generally assumes that everything is up to date after changing some data